docs/DQC_IFRS_0008/DQC_0008.md

```diff
@@ -1,18 +1,22 @@
 # Reversed Calculation
 Rule ID: DQC_0008   
 
-## Rule function
+## Rule description
 
-This rule evaluates whether a calculation relationship in the company's extension is a reversal of the calculation defined in the base taxonomy used for the filing. The rule checks if the element representing the calculation total and the element representing the component are a reversal of those same elements defined in the base taxonomy with the same calculation weight.  
+This rule evaluates whether a calculation relationship in the _report taxonomy_ is a reversal of the calculation defined in the base taxonomy used for the filing.  Where a concept appears as a contributing item to a total in the _base taxonomy_, the concept representing the total cannot appear as a contributing item to that concept in the _report taxonomy_, unless the calculation weight is reversed.
 
-This rule only compares the defined relationship between two elements joined by a single calculation arc in a company's calculation relationship and checks if the opposite of that relationship exists in the base taxonomy. Both elements in the calculation relationship must have a balance attribute of either debit or credit.  
+Note that this rule only checks for direct relationships between concepts, not all ancestors and descendents.
 
 ## Problem solved by the rule
 
 Users expect that the calculation relationships defined between elements in the base taxonomy can be used to identify the meaning of an element. For example when reporting using the IFRS Taxonomy, Mining Assets is a component of Property Plant and Equipment as Property Plant and Equipment  is comprised of Mining Assets, Land Buildings etc.  
  
 However, if the company changed the calculation relationship so that Property Plant and Equipment was a component of Mining Assets then a user of the data will have a different meaning of the concept of that reported by the preparer.  
 
+## Rule specification
```

Comment:
The rule implementation also has:

`and $extCalcSource != $extCalcTarget`

What is this guarding against?